code/API_definitions/qod-api.yaml

```diff
@@ -174,7 +174,7 @@ paths:
       description: |
         Important: this endpoint is to be implemented by the API consumer.
         The QoD server will call this endpoint whenever any network related event occurs.
-        Currently only SESSION_TERMINATED event is implemented. Any other network events are ignored.
+        Currently only QOS_STATUS_CHANGED event is implemented.
```

Comment:
I know I'm being a pedant here, but the OAS file is an API definition, not an implementation. So the description should read:
`Currently only the QOS_STATUS_CHANGED event is defined.`